### PR TITLE
Week 9 - Update UG documentation

### LINKED ISSUE #52: Week 9: Update UG - Features: Adding a person

### LINKED ISSUE #53: Week 9: Update UG - Features: Editing a person

### LINKED ISSUE #54: Week 9: Update UG - Command Summary: Add

### LINKED ISSUE #55: Week 9: Update UG - Command Summary: Edit

### LINKED ISSUE #57: Week 9: Update UG - Features: Add tag to a person

### LINKED ISSUE #58: Week 9: Update UG - Command Summary: Add Tag

### PR DESCRIPTION
**Pull request by ChongCheeKaiClarence**
Add and Edit commands do not use tags.

Tag add command now uses INDEX.

The box format might bug out.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -49,9 +49,6 @@ SoConnect is a **desktop app for managing contacts, optimized for use via a Comm
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
 * Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
```
Comment (florentianayuwono) [inline docs/UserGuide.md]:
```suggestion
  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family`, etc.
```

**Review by florentianayuwono — COMMENTED** [review]
Great work!

File: docs/UserGuide.md (hunk 1 above)
Comment (ChongCheeKaiClarence) [reply docs/UserGuide.md]: I have updated it.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -153,12 +143,16 @@ Example:
...
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A person can have any number of tags. Add as many as you want.
+</div>
 
-* `NAME` is the name of the contact to add `TAG` to.
+* Adds a `TAG` to the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 
 Example:
-* `tag add n/John Doe t/friend` adds the friend tag to John Doe.
+* `tag add 1 t/friend` adds the friend tag to John Doe.
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]: Specify that John Doe is index 1 on the currently displayed list? Specify that the `friend` tag already exists?

**Review by Eugene-Ong-W-X — COMMENTED** [review]
1 change to consider

File: docs/UserGuide.md (hunk 2 above)
Comment (ChongCheeKaiClarence) [reply docs/UserGuide.md]: Ok. I made changes.

**Comment by Eugene-Ong-W-X** [comment]
LGTM

**Pull request merged**